IdentityCore/src/oauth2/aad_v2/MSIDAADV2TokenResponseForV1Request.m

```diff
@@ -27,15 +27,9 @@
 
 @implementation MSIDAADV2TokenResponseForV1Request
 
-- (BOOL)initIdToken:(NSError **)error
+- (Class)tokenClaimsClass
```

Comment:
why doesn't this override claimsFromIdToken instead?

Reply:
fixed